decomposed/extensions/MDDE_LDM/Profile/Model/Forms/mdde_XmlExportConfiguration.xml

```diff
@@ -2,12 +2,22 @@
 <a:ObjectID>82A848B3-38C7-40F2-AB0F-ECE90F5DB908</a:ObjectID>
 <a:Name>mdde_XmlExportConfiguration</a:Name>

```

Comment:
Please rename this form to something like mdde_Configuration, since it's now more genericly, Not only about XML Export.